pykern/pkcli/github.py

```diff
@@ -49,6 +50,30 @@ def backup():
     pkdlog("DONE")
 
 
+def ci_check(repo, branch=None):
+    def _branch(r, name, reraise=True):
+        try:
+            return r.branch(name=name)
+        except github3.exceptions.NotFoundError:
+            if reraise:
+                raise
+            return None
+
+    r = _repo_arg(repo)
+    b = (
+        _branch(r, branch)
+        if branch
+        else _branch(r, "master", False) or _branch(r, "main")
+    )
+    s = b.commit.sha
+    c = [c.conclusion for c in b.commit.check_runs()]
+    if not c:
+        pkcli.command_error("No workflow runs for commit")
```

Comment:
Include branch name and sha. Watch for repetition when you do this to the next error message to.

Reply:
What do you mean by watch for repetition? Should I just print branch name and sha once before all the error checking?

Reply:
First step is add the branch and sha to all messages. Then, see what patterns repeat. Then, simplify the repetition.

Reply:
pksubprocess_test keeps randomly failing on this pr. Do you know what might be causing it?

```
=================================== FAILURES ===================================
_________________________ test_check_call_with_signals _________________________
Traceback (most recent call last):
  File "/home/runner/work/pykern/pykern/tests/pksubprocess_test.py", line 65, in test_check_call_with_signals
    pksubprocess.check_call_with_signals(cmd, output=o, msg=msg)
  File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/_pytest/python_api.py", line 966, in __exit__
    fail(self.message)
  File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/_pytest/outcomes.py", line 196, in fail
    raise Failed(msg=reason, pytrace=pytrace)
Failed: DID NOT RAISE <class 'RuntimeError'>
============================== 1 failed in 10.15s ==============================
error: FAILED=1 passed=37
```

Reply:
No, I don't. How would you instrument pksubrocess_test to debug this? How would you debug this more easily?

Reply:
I think the failing test is unrelated to the changes in this pr, so I can make an issue for it. This pr is ready for re-review. ea3c408a5e353e43282c3a154677ec387b309992